### PR TITLE
Raster style

### PR DESCRIPTION
**Pull request by JakobMiksch**
- adds styles to GeoServer in a programmatic way

@weskamm I thought about the SLD worker as well, but since this process is typically done during setup of the stack I decided to put the functionality in the `geoserver-init` service

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ mkdir klips-api-config
 cp klips-api/src/config/* klips-api-config/
 ```
 
+```
+
 developoment:
 
 ```shell
```
Comment (JakobMiksch) [inline README.md:22]:
```suggestion
```

developoment:

```shell
```

**Comment by weskamm** [comment]
LGTM, but why did you remove the `geoserver-init/010_create_workspace.js ` ?

**Review by weskamm — APPROVED** [review]

**Comment by JakobMiksch** [comment]
I just rename `geoserver-init/010_create_workspaces.js` to `geoserver-init/010_setup_geoserver.js`

**Comment by weskamm** [comment]
ahh all right then

**Pull request merged**